src/rdbms/mongoose_rdbms.erl

```diff
@@ -689,7 +703,9 @@ outer_op({sql_transaction, F}, State) ->
 outer_op({sql_dirty, F}, State) ->
     sql_dirty_internal(F, State);
 outer_op({sql_execute, Name, Params}, State) ->
-    sql_execute(outer_op, Name, Params, State).
+    sql_execute(outer_op, Name, Params, State);
+outer_op({sql_execute_measured, Name, Params, MeasureFun}, State) ->
+    MeasureFun(fun() -> sql_execute(outer_op, Name, Params, State) end).
```

Comment:
Test, what happens if MeasureFun crashes.

Reply:
I added a test to both cover `nested_op` and crash inside a wrapper. Is it sufficient?